packages/samples/react/src/style.scss

```diff
@@ -1,3 +1,7 @@
+:root {
+	--kolibri-root-font-size: 100px;
```

Comment:
@sdvg Die Änderung erschließt sich mir nicht. Hab ich was falsch verstanden?

Reply:
Was noch fehlt ist, dass wir auch die tatsächliche font-size analog dazu auf einen entsprechenden Wert setzen.
Die korrekten Werte müssten sein:

```
--kolibri-root-font-size: 100;
font-size: 625%;
```

Dann springt z.B. direkt in's Auge, dass die Build-ID viel zu groß ist, also vermutlich nicht die in Sass definierte `rem()`-Funktion verwendet. Solche Probleme sollten wir lösen.

![screenshot 2024-08-29-09 17 58@2x](https://github.com/user-attachments/assets/dc46c934-bac0-4c9f-bae9-4101556a0ddc)